_posts/2021-08-25-debezium-1-7-beta1-released.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Debezium 1.7.0.Beta1 Released
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, kafka ]
```

Comment:
here

Reply:
@rk3rn3r I want to see it here https://debezium.io/releases/1.7/ so you should update https://github.com/debezium/debezium.github.io/blob/develop/_data/releases/1.7/1.7.0.Beta1.yml#L4 as well

Reply:
oh, I think you mean `series.yml`

Reply:
It should be mentioned next to NATS as a key change in the description

Reply:
Ah, done now :)